### ISSUE TITLE
Simple issue fix....

### ISSUE THREAD

**Issue by k2-cell** [Ross1337/SMSBotBypass#47]
So after finally understanding it all and getting the bot into my server, i am stuck on the final part...... 
the node command......
I must copy and paste the location of the bot.js file for a command to even register, when i do "node /home/kali/SMSBotBypass/bots/discord/bot.js" and hit enter, it throws the error 

"internal/modules/cjs/loader.js:818
  throw err;
  ^

Error: Cannot find module 'discord.js'
"
Ive read the instructions almost a million times, doing my own research and still cant seem to find a fix, if you could please write me on telegram it would be much appreciated:) @Major

**Comment by Ross1337** [comment]
Have you done the npm i ?

**Comment by k2-cell** [comment]
> Have you done the npm i ?

Hey, thanks for the response. Yes I have done the npm I and the start. When I do start it says it started *1337 but no other command can be typed after that. That's when I start another terminal (with that one still open) and run through all the stuff again to start the bot.js. When I do the npm first time it gave 6 errors. Now only 3 after I ran clean. Think one error was error funding after running the npm I ???

**Comment by Ross1337** [comment]
What happens if you try to run a normal jsfile for testing using the same path ?

**Comment by k2-cell** [comment]
> What happens if you try to run a normal jsfile for testing using the same path ?

I'll be home in a hour to test this smart idea. Happen to have a specific jsfile for testing I should use? Or a suggested github download with plenty of json files to try? 
Just downloaded Linux Vm for this and try to get this working myself as I'm very curious on the benefits of this. Don't have a expert background with Linux but somewhat knowledgeable and problematic.

**Comment by Ross1337** [comment]
Linux is not needed, I made this bot on my windows computer (but it works on all plateforms) ; you can use a random nodejs file

**Comment by k2-cell** [comment]
> Linux is not needed, I made this bot on my windows computer (but it works on all plateforms) ; you can use a random nodejs file 

Welp..... that might be why it's a little more buggy for me hahah. Welp I will give it a shot on Windows and follow instructions exactly the same and it will probably work 😂

**Comment by Ross1337** [comment]
Normally it should work on all platforms

**Comment by k2-cell** [comment]
> Normally it should work on all platforms
okay so after i ran the test i got the call obv, but then i do "npm i"
and it comes back with the lines 

   
"
up to date, audited 421 packages in 957ms

28 packages are looking for funding
  run `npm fund` for details

3 moderate severity vulnerabilities

To address all issues, run:
  npm audit fix

Run `npm audit` for details.

"

**Comment by k2-cell** [comment]
i do 
"npm start"
comes back as

> SMS-Bypass@1.0.0 start
> node api

Express server started on *:1337
"
like its supposed to i assume

**Comment by k2-cell** [comment]
im retarded, i was trying to run node bot.js in the ~/SMSBotBypass/api instead of using ~/SMSBotBypass/bots/discord, after that the discord bot started right up :)

**Comment by k2-cell** [comment]
> Normally it should work on all platforms

okay so after using the call feature, the discord bot says that the calll has been sent, i receive no call and then the terminal where i ran the node.js it says logged in as discord sms#(my numbers of bot)
then under it (which appears ONLY after i use the discord bot call command)
"Error: connect ETIMEDOUT (MY IP:1337)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1138:16) {
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '(my IP)',
  port: 1337,
  config: {
    url: 'http://(my ip):1337/call/',
    method: 'post',
    data: 'password=(password)o=(test number)&user=%40kilo%237836&service=paypal&name=',                                             
    headers: {
      Accept: 'application/json, text/plain, */*',
      'Content-Type': 'application/x-www-form-urlencoded',
      'User-Agent': 'axios/0.21.1',
      'Content-Length': 96

**Comment by d0p3r0b0t** [comment]
like when you did the test call did it ring you number? if did he said thts good, now just figure why the api not talking to discord bot

**Comment by Ross1337** [comment]
This error came because you didn't modified the config file in the discord folder.

**Comment by k2-cell** [comment]
> This error came because you didn't modified the config file in the discord folder. 

Hey there. I did do this, maybe I did it wrong. Where it says the API irl I did put my IP:1337 maybe that's the issue? Not sure where to find my api irl.

**Comment by k2-cell** [comment]
> like when you did the test call did it ring you number? if did he said thts good, now just figure why the api not talking to discord bot
> 
> 

You figure it out? He stated it's in the discord config file and I am assuming what I said above this was where the issue lies. But Idk the API url? I even looked up online where tf to find it and it just gives me url for bot to join the channel? Maybe that's it now that I think about it?

**Comment by Ross1337** [comment]
Your api url is your public ip like 85.54.1.58:1337

**Comment by k2-cell** [comment]
> Your api url is your public ip like 85.54.1.58:1337

Hey boss, yeah that's what I had done initially and assumed it was wrong since you said that error gives it when you don't config the discord config. But I did do that. I'm still working on it and trying to find a way. Thank you for your help.

**Comment by Ross1337** [comment]
You can try to use your localhost ip (as the original config.js file). If it doesn't work, add me on telegram or discord

**Comment by k2-cell** [comment]
> You can try to use your localhost ip (as the original config.js file). If it doesn't work, add me on telegram or discord

The simple 127.0.0.1 ? I will give thay a try thank you. And if I can't figure out in next day then I'll for sure add you on discord. Here is mine kilo#7836

**Comment by k2-cell** [comment]
> You can try to use your localhost ip (as the original config.js file). If it doesn't work, add me on telegram or discord

Man you sure were right. After that I used the discord command, it gave me a call, told me I need to upgrade. So I upgraded assuming it would let me use the OTP voice bot. I did a test call !call (my number) default. It rang, answered, pure quietness for first 15 seconds then says "we're sorry, an application error has occurred,good bye". I assume it has to deal with a file and not twilio correct?
